tests/tests.rs

```diff
@@ -622,18 +622,6 @@ pub fn get_font_properties() {
     assert_eq!(properties.stretch, Stretch(1.0));
 }
 
-#[cfg(feature = "source")]
-#[test]
-pub fn get_font_data() {
```

Comment:
Why is this test being removed?
I re-added it locally and it passed.

Reply:
What OS are you on? It fails for me on Mac, iirc due to https://github.com/servo/font-kit/pull/232/files#diff-6ba3732b9cf93e63d757c9bbbe03032cb17ed4525b45153ec0217b7df271ebf1R132 which leads to `copy_font_data` call failing.

Reply:
Im on Linux

Reply:
Maybe it's unwise to remove the test. At the very least, we can cfg-gate it.